H/hidapi/build_tarballs.jl

```diff
@@ -34,11 +34,16 @@ cmake  \
     ${linuxflags} \
     ..
 
-cmake --build . --target install
+cmake --build . --parallel ${nproc} --target install
 install_license ../hidapi/LICENSE*.txt
 """
 
-platforms = supported_platforms()
+broken_platforms = Set([
```

Comment:
Updating libusb to a newer version should easily resolve this

Reply:
I must be doing something wrong: when I add `compat="^1.0.23"` (lowest available version of libusb_jll if I'm correct), I still get the error when I try to build `aarch64_unknown_freebsd`.

Reply:
What I'm suggesting is to build a new version of libusb in a separate PR: that should automatically bring the missing platforms without more effort.

Reply:
ah! sorry I misunderstood you. It's a bit late for me now, I'll do it later.